src/utils/buildInfo.ts

```
@@ -0,0 +1,241 @@
+}
+
+/**
+ * Obtient l'URL de téléchargement appropriée
+ */
+export function getDownloadUrl(distribution: BuildInfo['distribution'], repo: string = DEFAULT_GITHUB_REPO): string {
+  switch (distribution) {
+    case 'microsoft-store':
+      return 'ms-windows-store://pdp/?productid=PRODUCT_ID'; // À remplacer par l'ID réel
+    case 'github':
+    case 'portable':
+    default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)